apps/extension/src/App/Settings/Currency.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from "react";
+
+import { GapPatterns, Stack, TickedRadioList } from "@namada/components";
+import { FiatCurrency } from "@namada/types";
+import { PageHeader } from "App/Common";
+
+import { ExtensionKVStore } from "@namada/storage";
+import { KVPrefix } from "router/types";
+import { LocalStorage } from "storage";
+import browser from "webextension-polyfill";
+
+const localStorage = new LocalStorage(
```

Comment:
Not sure if we use this anywhere else in the extension, but we might make it a hook or sth :)